docs/wiki/featurex/volume.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: wiki
+title: Volume Toggle
+description: Keybind to toggle volume of game and music.
+group: feature
+category: general
+parent: wiki
+mod: acex
+version:
+  major: 3
+  minor: 2
+  patch: 0
+---
+
+## 1. Overview
+
+Adds a keybind to toggle volume of game and music. It's quite useful for when you're briefing in heavy rain, or if you're in an obnoxiously loud car.
```

Comment:
Maybe write the part about the keybind differently, eg. "Adds the possibility of setting a keybind to toggle ..."?